data/json/monsters/insect_spider.json

```
@@ -1356,5 +1356,34 @@
     "death_function": [ "NORMAL" ],
     "special_attacks": [ { "type": "leap", "cooldown": 4, "max_range": 4, "allow_no_target": true }, [ "EAT_CROP", 120 ] ],
     "flags": [ "SEES", "HEARS", "SMELLS", "CLIMBS", "LARVA", "STUMBLES", "PATH_AVOID_FIRE" ]
+  },
+  {
+    "id": "mon_mantis",
+    "type": "MONSTER",
+    "name": { "str": "giant preying mantis" },
```

```suggestion
    "name": { "str": "giant preying mantis", "str_pl": "giant preying mantises" },
```
Or mantes? I don't know.